src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -834,6 +1109,99 @@ private void AddMetaKeyValues(int i, DataViewSchema.Annotations.Builder builder)
             }
 
             protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, bool> activeOutput, out Action disposer) => _parent.GetGetterCore(input, iinfo, out disposer);
+
+            private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, string srcVariable, string dstVariable)
+            {
+                string opType;
+                string castOutput;
+                OnnxNode castNode;
+                OnnxNode murmurNode;
+
+                opType = "MurmurHash3";
+                string murmurOutput = ctx.AddIntermediateVariable(_dstTypes[iinfo], "MurmurOutput");
+                var srcShape = ctx.RetrieveShapeOrNull(srcVariable);
+                var srcType = _srcTypes[iinfo].GetItemType().RawType;
+
+                // Numeric input types are limited to those supported by the Onnxruntime MurmurHash operator, which currently only supports
+                // uints and ints. Thus, ulongs, longs, doubles, floats, and booleans are not supported.
+                if (srcType == typeof(ushort) || srcType == typeof(short) || srcType == typeof(sbyte) ||
+                    srcType == typeof(byte) || srcType == typeof(bool))
+                {
+                    castOutput = ctx.AddIntermediateVariable(NumberDataViewType.UInt32, "CastOutput", true);
+                    castNode = ctx.CreateNode("Cast", srcVariable, castOutput, ctx.GetNodeName(opType), "");
+                    castNode.AddAttribute("to", NumberDataViewType.UInt32.RawType);
+                    murmurNode = ctx.CreateNode(opType, castOutput, murmurOutput, ctx.GetNodeName(opType), "com.microsoft");
+                }
+                else if (srcType == typeof(uint) || srcType == typeof(int) || srcType == typeof(ReadOnlyMemory<char>))
+                {
+                    murmurNode = ctx.CreateNode(opType, srcVariable, murmurOutput, ctx.GetNodeName(opType), "com.microsoft");
+                }
+                else
+                {
+                    return false;
+                }
+
+                murmurNode.AddAttribute("positive", 1);
+                var seed = _parent._columns[iinfo].Seed;
+                if (_parent._columns[iinfo].UseOrderedHashing)
+                {
+                    if (srcShape[1] > 1) return false;
```

Comment:
>1 [](start = 33, length = 1)

Doesn't this need to be `srcShape[0]`? 

I think it would be better to look at `_srcTypes[iinfo]` to check whether the input is a vector or not.

Reply:
srcShape[0] is the dimension reserved for batches, so it's always -1.

Reply:
Changed it tho!